src/components/sbb-header/sbb-header.e2e.ts

```diff
@@ -10,4 +10,105 @@ describe('sbb-header', () => {
     element = await page.find('sbb-header');
     expect(element).toHaveClass('hydrated');
   });
+
+  it('should be fixed on scroll', async () => {
+    page = await newE2EPage();
+    await page.setContent(`
+        <sbb-header></sbb-header>
+        <div style="height: 2000px;"></div>
+    `);
+    element = await page.find('sbb-header');
+
+    await page.evaluate(() => window.scrollTo({ top: 200 }));
+    await page.waitForChanges();
+    expect(element).toEqualHtml(`
```

Comment:
I suggest to only check data attributes instead of the whole html tree, or even check with computedStyle if the header really is showing or if it is hidden. Comment also valid for following assertions.